### ISSUE TITLE
Footers not skipped in systemctl list-unit-files parser

### ISSUE THREAD

**Issue by kellyjonbrazil** [kellyjonbrazil/jc#39]
Footer lines not being skipped
```
...
  {
    "unit": "systemd-tmpfiles-clean.timer",
    "file": "static"
  },
  {
    "unit": "246",
    "file": "unit",
    "state": "files"
  }
]
```

**Comment by kellyjonbrazil** [comment]
Looks like the issue is not with the parser, but with the Magic syntax.
```
jc -p systemctl list-unit-files
```
Looks like the Magic syntax is picking the wrong parser, probably plain `systemctl`.

**Comment by kellyjonbrazil** [comment]
@philippeitis it looks like this issue was introduced https://github.com/kellyjonbrazil/jc/pull/32.

**Comment by kellyjonbrazil** [comment]
fix merged into v1.8.1
